CAP/gap/ToolsForCategories.gd

```diff
@@ -95,3 +95,17 @@ DeclareGlobalFunction( "ListKnownCategoricalProperties" );
 DeclareGlobalFunction( "CAP_MergeRecords" );
 
 DeclareGlobalFunction( "HelpForCAP" );
+
+#! @Arguments category[, operation]
+#! @Description
+#! Prints statistics for all caches in <A>category</A>.
+#! If <A>operation</A> is given (as a string), only statistics
+#! for the given operation cache is stored.
+DeclareGlobalFunction( "CachingStatistic" );
+
+
+#! @Arguments category
+#! @Description
+#! Displays statistics for all caches in <A>category</A>.
+#! in a Browse window.
```

Comment:
Maybe it is a good idea to give a short description of the 4 columns:
status, hits, misses, stored

Reply:
Good idea, will do so.